docs/DeveloperGuide.md

```diff
@@ -312,32 +312,157 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the System is the `TutorTrack` and the Actor is the `Tutor`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case:** UC01 - View all students.<br>
+**Preconditions:** Connection is stable. 
```

Comment:
> **Preconditions:** Connection is stable. 

sir this is a local application 😂 maybe we can remove the preconditions for now

Reply:
Yup same comment here. But honestly, for now, I think this is good enough to merge since we don't really have that concrete of use cases planned out anyways.